site/en/blog/web-custom-formats-for-the-async-clipboard-api/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Web custom formats for the Async Clipboard API
+subtitle: >
+  Web custom formats let websites read and write arbitrary unsanitized payloads using a standardized
+  format applications can opt in to if they wish to support such payloads.
+authors:
+  - thomassteiner
+date: 2022-07-25
+---
+
+Up until now, the [Async Clipboard API](/async-clipboard/) supported a limited set of MIME types to
```

Comment:
By the way, did you plan to update https://web.dev/async-clipboard/ to include a link to this dcc article as well?

Reply:
Good catch!